README.md

```diff
@@ -71,6 +71,16 @@ regions.each do |region|
 end
 ```
 
+If you want to specify roles for a subset of instances in the autoscaling group, you can use
+`:partial_roles`.  These will be spread out 1 per server:
```

Comment:
> These will be spread out 1 per server:

One per server or 1 (N) per group?

Reply:
It's pretty basic as it is now, it iterates over all the partial_roles and adds them to one `partial_queue` array.  It then shifts one element at a time as it registers the servers.  If there are more partial_roles than servers, they will be dropped with a warning.